test/atm/global-workflow/setup_workflow_exp.sh

```diff
@@ -12,7 +12,7 @@ starttype='warm'
 gfscyc='4'
 resdet='48'
 resens='48'
-nens=0
+nens=10
```

Comment:
Since we're not testing science here, could `nens` be reduced to 3?

Reply:
Good idea.   I set `nens=3` and reran the atm setup, ens prep, ens run, and ens post tests.   All ran to completion.   `test_gdasapp_atm_jjob_ens_run` ran faster:   234.19 seconds on Hera and 235.23 seconds on Orion.  Change committed to `feature/ctest_atm_letkf` at [fc882ce](https://github.com/NOAA-EMC/GDASApp/commit/fc882ce424800e245f849c0f483d8c59660ea784).